build.sbt

```diff
@@ -20,21 +20,20 @@ import uk.gov.hmrc.DefaultBuildSettings.{addTestReportOption, defaultSettings, s
 import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
 
 val appName = "vat-correspondence-details-frontend"
-
 val bootstrapPlayVersion       = "5.12.0"
-val playFrontendHmrc           = "1.4.0-play-26"
-val playPartialsVersion        = "8.2.0-play-26"
-val authClientVersion          = "3.0.0-play-26"
-val playLanguageVersion        = "5.1.0-play-26"
-val scalaTestPlusVersion       = "3.1.3"
-val hmrcTestVersion            = "3.9.0-play-26"
-val scalatestVersion           = "3.0.8"
+val playFrontendHmrc           = "0.94.0-play-28"
```

Comment:
Is there any reason why you downgraded play-frontend-hmrc? Could you please upgrade it back to the version it was on before?